test/metabase/db/schema_migrations_test.clj

```diff
@@ -1000,3 +1000,49 @@
                 pg-field-3-id :type/Text
                 mysql-field-1-id :type/SerializedJSON
                 mysql-field-2-id :type/Text))))))))
+
+(deftest migrate-field-json-unfolding-type-test
+  (testing "Migration v47.00-003: set base-type to type/JSON for JSON database-types for postgres and mysql"
+    (impl/test-migrations ["v47.00-003"] [migrate!]
+      (let [[enabled-db-id
+             enabled-by-default-db-1-id
+             enabled-by-default-db-2-id
+             disabled-db-id] (t2/insert-returning-pks! Database [{:name    "enabled"
+                                                                  :engine  "postgres"
+                                                                  :details {:json-unfolding true}}
```

Comment:
worth adding a case where `json-unfolding` key is not present? Not sure if it could lack that key in the wild, but would be nice to know if the migration can handle that case.

Reply:
Isn't that case handled by lines 1013-1015?https://github.com/metabase/metabase/pull/28742/files/c9c911bace9294a62e63d816fce42a35adae091d#diff-41093075df90623837726c733afc60b57899a66d6e9878b4250413b74f045f67R1013-R1015

Reply:
ah nice, sorry I didn't see that.